NetKVM/wlh/ParaNdis6_RSS.cpp

```diff
@@ -65,32 +72,31 @@ static VOID InitRSSCapabilities(PARANDIS_ADAPTER *pContext)
     RSSCapabilities->Header.Type = NDIS_OBJECT_TYPE_RSS_CAPABILITIES;
     RSSCapabilities->Header.Revision = NDIS_RECEIVE_SCALE_CAPABILITIES_REVISION_1;
     RSSCapabilities->Header.Size = NDIS_SIZEOF_RECEIVE_SCALE_CAPABILITIES_REVISION_1;
-    RSSCapabilities->CapabilitiesFlags =    NDIS_RSS_CAPS_MESSAGE_SIGNALED_INTERRUPTS |
-                                        NDIS_RSS_CAPS_CLASSIFICATION_AT_ISR |
-                                        NdisHashFunctionToeplitz;
+    RSSCapabilities->CapabilitiesFlags = NDIS_RSS_CAPS_MESSAGE_SIGNALED_INTERRUPTS |
+                                         NDIS_RSS_CAPS_CLASSIFICATION_AT_ISR | NdisHashFunctionToeplitz;
     if (pContext->bRSSSupportedByDevice)
     {
         ULONG flags = 0;
-        flags |= (pContext->DeviceRSSCapabilities.SupportedHashes & VIRTIO_NET_RSS_HASH_TYPE_TCPv4) ?
-            NDIS_RSS_CAPS_HASH_TYPE_TCP_IPV4 : 0;
-        flags |= (pContext->DeviceRSSCapabilities.SupportedHashes & VIRTIO_NET_RSS_HASH_TYPE_TCPv6) ?
-            NDIS_RSS_CAPS_HASH_TYPE_TCP_IPV6 : 0;
-        flags |= (pContext->DeviceRSSCapabilities.SupportedHashes & VIRTIO_NET_RSS_HASH_TYPE_TCP_EX) ?
-            NDIS_RSS_CAPS_HASH_TYPE_TCP_IPV6_EX : 0;
+        flags |= (pContext->DeviceRSSCapabilities.SupportedHashes & VIRTIO_NET_RSS_HASH_TYPE_TCPv4) ? NDIS_RSS_CAPS_HASH_TYPE_TCP_IPV4
```

Comment:
1. More than 120 lines
2.  In any case let's check if possible to keep the expressions after "?" on one line
We can disable formating for now

Reply:
It behaves this way because we set ContinuationIndentWidth to 100. This allows us to break ColumnLimit where necessary. If we set it to 0, it could affect other parts of the code.

Reply:
Anyway this lines not far from 120, it has from 120 to 125 symbols here